index.html

```diff
@@ -1,1074 +1,1650 @@
-    and <code>TYPE_BACK_FORWARD</code> as listed below.</p></dd>
-  <dt><dfn>TYPE_RELOAD</dfn></dt>
-    <dd><p>Navigation through
-    the reload operation or the
-    <a href="https://www.w3.org/TR/html5/browsers.html#dom-location-reload">location.reload()</a>
-    method.</p></dd>
-  <dt><dfn>TYPE_BACK_FORWARD</dfn></dt>
-   <dd><p>Navigation
-  through a <a href="https://www.w3.org/TR/html5/browsers.html#traverse-the-history">history traversal</a> operation.</p></dd>
-  <dt><dfn>TYPE_RESERVED</dfn></dt>
-  <dd><p>Any navigation types not defined by values above.</p></dd>
-
-<dt><dfn>type</dfn></dt>
-<dd>
-
-<p>This attribute must return the type of the last non-redirect <a
-href="https://www.w3.org/TR/html5/browsers.html#navigate">navigation</a>
-in the current browsing context. It must have one of the following <a
-href="#dom-performancenavigationtiming-type">navigation type</a> values. </p>
-
-<p class="note">Client-side redirects, such as those using <a href="https://www.w3.org/TR/html5/document-metadata.html#attr-meta-http-equiv-refresh">the Refresh pragma directive</a>,
-are not considered HTTP redirects <a href="https://www.w3.org/TR/html5/infrastructure.html#concept-http-equivalent-codes">or equivalent</a> by this spec.
-In those cases, the <a href="#dom-performancenavigationtiming-type">type</a> attribute should return appropriate value,
-such as <code>TYPE_RELOAD</code> if reloading the current page, or
-<code>TYPE_NAVIGATE</code> if navigating to a new URL.</p>
-
-</dd>
-<dt><dfn>redirectCount</dfn></dt>
-<dd>
-
-<p>This attribute must return the number of redirects since the last
-non-redirect navigation under the current browsing context. If there is no
-redirect or there is any redirect that is not from the same
-<a href="https://tools.ietf.org/html/rfc6454">
-origin</a> as the destination document, this attribute must return zero. </p>
-</dd>
-</dl>
-
-</section>
-
-<section data-dfn-for="Performance">
-<h3>Extensions to the <code>Performance</code> interface</h3>
-<pre class="idl">
-[Exposed=Window]
-partial interface Performance {
-  [SameObject]
-  readonly attribute PerformanceTiming timing;
-  [SameObject]
-  readonly attribute PerformanceNavigation navigation;
-};
-</pre>
-<dl>
-
-<dt><dfn>timing</dfn></dt>
-<dd>
-<p>
-  The <code>timing</code> attribute
-  represents the timing information related to the browsing contexts since the
-  last non-redirect navigation. This attribute is defined by the <code>PerformanceTiming</code> interface.
-</p>
-</dd>
-<dt><dfn>navigation</dfn></dt>
-<dd>
-<p>
-  The <code>navigation</code> attribute is defined by the <code>PerformanceNavigation</code> interface.
-</p>
-</dd>
-</dl>
-</section>
-
-</section>
-
-<section id="acknowledgements" class="appendix">
-  <h2>
-    Acknowledgments
-  </h2>
-  <p>Thanks to
-  Anne Van Kesteren,
-  Arvind Jain,
-  Boris Zbarsky,
-  Jason Weber,
-  Jonas Sicking,
-  James Simonsen,
-  Karen Anderson,
-  Nic Jansma,
-  Philippe Le Hegaret,
-  Steve So
```

Comment:
This seems like an editorial bug, s/`href="https://heycam.github.io/!WEBIDL/#idl-enums"`/`data-cite="!WEBIDL/#idl-enums"` ?